.dockerignore

```diff
@@ -3,13 +3,13 @@
 
 # Allow files and directories
 !/requirements.txt
-!/requirements-backend.txt
-!/requirements-processing.txt
-!/requirements-submissions.txt
+!/python_dependecies/backend/requirements.txt
```

Comment:
Typo: `python_dependecies` -> `python_dependencies`

Reply:
Also, any reason for the underscore? Normally dashes are standard.

Reply:
I see a mix of `_` and `-` used throughout our naming schema for files and directories. `_` are python friendly so it's my default.